openssl-dynamic/src/main/c/jnilib.c

```diff
@@ -411,7 +411,6 @@ jint JNI_OnLoad_netty_tcnative(JavaVM* vm, void* reserved) {
         return JNI_ERR;
     }
 #else
-    (void)parsePackagePrefix;
```

Comment:
This was necessary because if `TCN_NOT_DYNAMIC` is defined, then there are no refs to `parsePackagePrefix` which  results in a compilation failure (we use `[-Werror,-Wunused-function]`).   Wrapping that function in the ifndef makes the compiler complain about no refs to `netty_internal_tcnative_util_strstr_last` and `netty_internal_tcnative_util_strndup`.   At that point I realized it was going to complicate the code aloot to find all transitive calls of `parsePackagePrefix` and check if they only had one reference.  This line was the cleanest way I could come up with. 

I also tried making the function static, but I got more compilation errors: 

> error: static declaration of 'netty_internal_tcnative_util_prepend' follows non-static declaration

I was hoping that if the methods were static, the compiler would just ignore them, but I don't know what C compilers do nowadays.